packages/trees/frontend/sirius-components-trees/src/modals/upload-document/UploadDocumentModal.tsx

```diff
@@ -0,0 +1,176 @@
+/*******************************************************************************
```

Comment:
Remove the original in `packages/core/frontend/sirius-components/src/modals/upload-document`.